File: 110algoritmos/exercicios011/ex011-Elixir/ex011.exs
l = IO.gets("Digite a largura da parede: ")
{largura, _} = Float.parse(l)

a = IO.gets("Digite a altura da parede: ")
{altura, _} = Float.parse(a)

area = altura * largura
tinta = area / 2

IO.puts("\nA parede tem dimensoes #{largura} x #{altura}")
IO.puts("A parede tem area de #{area} metros quadrados")
IO.puts("Sao necessario #{tinta} litros de tinta para pinta-la.\n")

File: 110algoritmos/exercicios011/ex011-Csharp/ex011.cs
using System;

class Program {
    static void Main() {
        double largura, altura, area, tinta;

        Console.WriteLine("Digite a largura da parede: ");
        largura = double.Parse(Console.ReadLine());
        
        Console.WriteLine("Digite a altura da parede: ");
        altura = double.Parse(Console.ReadLine());

        area = altura * largura;
        tinta = area/2;

        Console.WriteLine("A area eh igual a: " + area +
                         ".\nSao necessarios " + tinta + " litros de tinta para pintar."
                            
        );       
    }
}